charts/portworx/v2.8.1/templates/portworx-csi.yaml

```diff
@@ -54,6 +103,9 @@ metadata:
   name: px-csi-ext
   namespace: kube-system
 spec:
+  selector:
```

Comment:
This block should actually be a Deployment, not a stateful set - 
https://github.com/portworx/px-installer/blob/master/px-spec-websvc/templates/template_portworx.gtpl#L676